docs/installation.rst

```diff
@@ -532,6 +532,10 @@ These platforms have been reported to work at the versions mentioned.
 +----------------------------------+------------------------------+--------------------------------+-----------------------+
 | Raspbian Stretch                 | 2.7, 3.5                     | 4.0.0                          |arm                    |
 +----------------------------------+------------------------------+--------------------------------+-----------------------+
+| Raspberry Pi OS                  | 2.7                          | 6.2.2                          |arm                    |
+|                                  +------------------------------+--------------------------------+-----------------------+
+|                                  | 3.6, 3.7, 3.8, 3.9           | 8.2.0                          |arm                    |
```

Comment:
```suggestion
| Raspberry Pi OS                  | 3.6, 3.7, 3.8, 3.9           | 8.2.0                          |arm                    |
|                                  +------------------------------+--------------------------------+                       |
|                                  | 2.7                          | 6.2.2                          |                       |
```
Suggestion to swap the rows so that the latest version of Pillow is at the top, and to merge the two 'arm' cells